web/client/components/misc/EmptyView.jsx

```diff
@@ -0,0 +1,39 @@
+ /*
+  * Copyright 2017, GeoSolutions Sas.
+  * All rights reserved.
+  *
+  * This source code is licensed under the BSD-style license found in the
+  * LICENSE file in the root directory of this source tree.
+  */
+const React = require('react');
+
+const FullWidthIcon = require('./FullWidthGlyph');
+
+/**
+ * A component to display an empty page.
+ * It allows to display a big icon with some information centered. As an option some additiona content can be provided via props. (e.g. add buttons, tools or info)
+ * It is very similar to material design concept for [empty state](https://material.io/guidelines/patterns/empty-states.html)
+ *
+ * @class EmptyView
+ * @memberof components.misc
+ * @param  {Number} [opacity=0.45]      The opactity
+ * @param  {String} [glyph="info-sign"] The icon glyph
+ * @param  {string|node} [title]               The title of the advice to display
+ * @param  {string|node} [description]         The description to display
+ * @param  {string|node} [content]             Additional content for the empty view (e.g. buttons...)
+ */
+module.exports = ({
+        opacity= 0.45,
```

Comment:
css?

Reply:
I will place it in css. Anyway I should keep the possibility to override the main div style (and then the stylesheet opacity) to allow quick fix of edge cases.

Reply:
Having a style property (with empty defaults) is ok